site/plugin.md

```diff
@@ -33,32 +33,48 @@ The Weave plugin actually provides *two* network drivers to Docker
 one named `weave` that can only work with one (like Docker's overlay
 driver).
 
+Docker supports creating multiple networks via the plugin, although
+Weave Net provides only one network underneath. So long as the IP
+addresses for each network are on different subnets, containers on one
+network will not be able to communicate with those on a different network.
+
 ### `weavemesh` driver
 
 * Weave handles all co-ordination between hosts (referred to by Docker as a "local scope" driver)
-* Supports only a single network (we create one named `weave` for you automatically)
 * Uses Weave's partition tolerant IPAM
+* User must pick different subnets if creating multiple networks.
+
+We create a network named `weave` for you automatically, using the
+[default subnet](ipam.html#subnets) set for the `weave` router.
+
+To create additional networks using the `weavemesh` driver, pick a
+different subnet and run this command on all hosts:
 
-If you do create additional networks using the `weavemesh` driver,
-containers attached to them will be able to communicate with
-containers attached to `weave`; there is no isolation between those
-networks.
+    $ docker network create --driver=weavemesh --ipam-driver=weavemesh --subnet=<subnet> <network-name>
```

Comment:
Isn't it a good idea to mention `--ip-range` here?
I got IPs assigned from default subnet without explicit ip-range setting.